dashboard/app/views/maker/login_code.html.haml

```diff
@@ -1,10 +1,10 @@
-%h1= 'Login to the Maker App with Google'
+%h1= I18n.t('maker.google_oauth.login_with_google')
 
-%p Please enter the code provided when you logged in via Google on your default browser
+%p= I18n.t('maker.google_oauth.enter_code')
 
 %input#input-text{:type=>"text"}
 
-%button#btn-submit{ type: 'submit' } Submit
+%button#btn-submit{ type: 'submit' }= I18n.t(:submit)
```

Comment:
sorry, more nits: can we make these HTML elements railsy (like we did in `display_code.html.haml`)? specifically, the `%input` and this `%button`

Reply:
This happens in the next PR: https://github.com/code-dot-org/code-dot-org/tree/google-maker-decrypt
Should I move those changes into this PR?

Reply:
woops, forgot about that! no need to move them, that's perfect